src-docs/src/views/filter_group/filter_group_multi.js

```diff
@@ -82,7 +82,7 @@ export default () => {
           noMatchesMessage="No filters found"
         >
           {(list, search) => (
-            <div style={{ width: 300 }}>
+            <div>
               <EuiPopoverTitle paddingSize="s">{search}</EuiPopoverTitle>
               {list}
             </div>
```

Comment:
Super optional, I'm curious if this could just be a fragment

```suggestion
            <>
              <EuiPopoverTitle paddingSize="s">{search}</EuiPopoverTitle>
              {list}
            </>
```